client/src/components/DatasetInformation/DatasetInformation.vue

```diff
@@ -100,7 +100,10 @@ export default {
     },
     methods: {
         bytesToString(raw_size) {
-            return bytesToString(raw_size, false);
+            return Utils.bytesToString(raw_size, false);
+        },
+        prefixedUrl(url) {
+            return window.options?.root.replace(/\/$/, "") + url;
```

Comment:
Relying on `window.options` seems to be unnecessary. Please note that there are utilities in `utils/redirect` which we could use here.

Reply:
I forgot to mention that I extracted this non-viz bug into https://github.com/galaxyproject/galaxy/pull/16301. I added your suggestion there.

Reply:
Ah cool!! Didnt realise there were helper functions. Do you think [withPrefix](https://github.com/galaxyproject/galaxy/blob/faa6b4c82d4478275ed15a94e826519e625db1c0/client/src/utils/redirect.ts#L42-L46) is more relevant than [prependPath](https://github.com/galaxyproject/galaxy/blob/faa6b4c82d4478275ed15a94e826519e625db1c0/client/src/utils/redirect.ts#L19-L22)?

Reply:
@martenson Perfect!! Then I will remove these changes in this PR and refer the new PR in the commit message. Does it sound reasonable to you?

Reply:
awesome, sorry for just pinging the PR without proper comment